app/src/main/java/doit/study/droid/quiz/QuizView.kt

```diff
@@ -0,0 +1,22 @@
+package doit.study.droid.quiz
+
+import doit.study.droid.R
+
+
+data class QuizView (
```

Comment:
`View` postfix is typically used for UI components, and this one is a data class. It would be better to call it just `Quiz`